server/repository/assessment/readTableData.ts

```diff
@@ -0,0 +1,58 @@
+import { BaseProtocol, DB, Schemas } from '@server/db'
+import { Objects } from '@core/utils'
+import { Assessment, Cycle, NodeValue } from '@meta/assessment'
+import { CountryIso } from '@meta/area'
+
+// TODO: Update return type from <any>
+
+export const readTableData = async (
+  props: {
+    assessment: Assessment
+    countryIso: CountryIso
+    cycleName: string
+    tableName: string
+  },
+  client: BaseProtocol = DB
+): Promise<{
+  data: Record<CountryIso, Record<string, NodeValue>>
```

Comment:
Yes, I am not 100% sure this is the final type but I can add it already there
Should it be instead of NodeValue something with Node which has the value?

Reply:
we only need the node